source/hardware/controllers/akai_apc_mini.rst

```diff
@@ -0,0 +1,20 @@
+AKAI APC Mini
+===================
+
+The *AKAI APC Mini* is a controller for a 64 Samplers deck.
+
+-  `Manufacturer’s product
+   page <https://www.akaipro.com/apc-mini>`__
+
+Mapping description
+-------------------
+
+-  The 64 pads are mapped directly to the 64 Samplers available in the extended skins, if the Skin has less Samplers, the non matching buttons will simply not work.
+-  All buttons lights up in yellow when a sample is configured in the corresponding slot
+-  When pressed and while playing, they will linght green.
+-  When stopped, they will light in red.
```

Comment:
```suggestion
-  When stopped, they will light up in red.
```

Reply:
Btw, the LED states would probably make sense to be put into a table for better readability.

Reply:
Correct, will do

Reply:
Done!